### ISSUE TITLE
Feature request add an option to pass activation function to TTA

### ISSUE THREAD

**Issue by ternaus** [BloodAxe/pytorch-toolbelt#26]
https://github.com/BloodAxe/pytorch-toolbelt/blob/develop/pytorch_toolbelt/inference/tta.py#L135

In many cases averaging logits works worse than averaging probabilities => would be nice to be able to pass user-defined activation function. For example softmax or sigmoid.

**Comment by ternaus** [comment]
A better option would be to apply desired functions on the model itself:

https://github.com/BloodAxe/Kaggle-2019-Blindness-Detection/blob/master/retinopathy/inference.py#L159
